src/Components/SmartComponents/CVEs/CVEsTable.js

```diff
@@ -83,6 +83,7 @@ const CVEsTableWithContext = ({ context, header, entity }) => {
                         }
                         ouiaId={'cves-table'}
                         isStickyHeader
+                        variant="compact"
```

Comment:
PF provides `TableVariant` constants
 
`import { TableVariant } from '@patternfly/react-table';` 
for this case `TableVariant.compact`